src/gfx/lib.rs

```diff
@@ -89,20 +89,11 @@ impl<D: device::Device> Graphics<D> {
 
     /// Draw a ref batch.
     pub fn draw<'a, T: shade::ShaderParam>(&'a mut self,
-        batch: &'a batch::RefBatch<T>, data: &'a T, frame: &Frame) {
+                batch: &'a batch::RefBatch<T>, data: &'a T, frame: &Frame)
+                -> Result<(), DrawError> {
         self.renderer.draw(&(batch, data, &self.context), frame)
     }
 
-    /// Draw a ref batch with instancing
-    pub fn draw_instanced<'a, T: shade::ShaderParam>(&'a mut self,
-                          batch: &'a batch::RefBatch<T>,
-                          data: &'a T,
-                          count: u32,
-                          vertex_offset: u32,
-                          frame: &Frame) {
-        self.renderer.draw_instanced((batch, data, &self.context), count, vertex_offset, frame)
-    }
-
```

Comment:
It's in the `Graphics` struct, which is a simple wrapper around `Renderer` and `Device` for single-threaded mode. It's only a convenience shortcut for simple applications. Once you want anything more advanced, you get rid of `Graphics` or just use `Renderer` directly (it's a public field). Sounds good?